#! /bin/sh
patch -p1 -l -f $* < $0
exit $?

From 50e65b8425e2dece5dc791269bc564168bf819e1 Mon Sep 17 00:00:00 2001
From: Andre Heinecke <aheinecke@intevation.de>
Date: Tue, 16 Oct 2018 14:14:29 +0200
Subject: [PATCH] Pass lang correctly to config dialog

* src/addin-options.cpp (open_gpgolconfig): Pass lang.
---
 src/addin-options.cpp | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/src/addin-options.cpp b/src/addin-options.cpp
index d82b095..137d880 100644
--- a/src/addin-options.cpp
+++ b/src/addin-options.cpp
@@ -185,6 +185,9 @@ open_gpgolconfig (LPVOID arg)
   args.push_back (std::string("--gpgol-version"));
   args.push_back (std::string(VERSION));

+  args.push_back (std::string ("--lang"));
+  args.push_back (std::string (gettext_localename ()));
+
   auto ctx = GpgME::Context::createForEngine (GpgME::SpawnEngine);
   if (!ctx)
     {
--
2.11.0
